.golangci.yml

```diff
@@ -38,14 +38,11 @@ linters:
     - depguard
     - dupl
     - dupword
-    - errcheck
     - errchkjson
     - errname
     - errorlint
     - gocritic
-    - godot
     - gofmt
-    - gofumpt
```

Comment:
Why disable this? Is it just for the perm octal? Then we can keep it, it looks useful